### ISSUE TITLE
Error when using `:where` - `subselects_1.subselects[name]`

### ISSUE THREAD

**Issue by wSedlacek** [GoogleChromeLabs/critters#90]
When using the `:where` selector with `critters@0.0.14` as part of `@angular-devkit/build-angular@13.0.4` like in the following selector:
```scss
/* Remove all the styles of the "User-Agent-Stylesheet", except for the 'display' property */
*:where(:not(iframe, canvas, img, svg, video):not(svg *, symbol *)) {
  all: unset;
  display: revert;
}
```

I get the following warning in my console
```
Warning: 1 rules skipped due to selector errors:
  *:where(:not(iframe, canvas, img, svg, video):not(svg *, symbol *)) -> subselects_1.subselects[name] is not a function
```

And the selector does not get inlined.

**Comment by wSedlacek** [comment]
Recreation:
https://stackblitz.com/edit/node-yfjhw4?file=index.ts
`npm run start`

**Comment by developit** [comment]
This is fixed by fb55/css-select#582, which was merged on Oct 15th but has not yet been released.
